nano/node/stats.hpp

```diff
@@ -292,6 +293,10 @@ class stat final
 		invalid_node_id_handshake_message,
 		outdated_version,
 
+		// tcp
+		tcp_accept_success,
+		tcp_accept_failure,
```

Comment:
Can these stats be checked in the socket test at all?

Reply:
Added checks. It's a bit icky to check stats that's updated async', but it seems to work locally. Will revisit if tests starts failing on the added checks.